hazelcast/src/main/java/com/hazelcast/monitor/impl/NearCacheStatsImpl.java

```diff
@@ -41,6 +41,16 @@
             newUpdater(NearCacheStatsImpl.class, "evictions");
     private static final AtomicLongFieldUpdater<NearCacheStatsImpl> EXPIRATIONS =
             newUpdater(NearCacheStatsImpl.class, "expirations");
+    private static final AtomicLongFieldUpdater<NearCacheStatsImpl> PERSISTENCE_COUNT =
```

Comment:
Hmm, I had concurrent writers in the first approach, but had to change this to a single writer, due to Jaromir's concern about concurrent disk writes. Actually the only new field with a FieldUpdater is `PERSISTENCE_COUNT` and I used it to have the convenient `PERSISTENCE_COUNT.incrementAndGet(this)` for an atomic increment.